INSTRUCTIONS.md

```diff
@@ -0,0 +1,15 @@
+In order to deploy the required cluster resources:
+./bootstrap.sh
+
+In order to check whether the todoapp server pod is running:
+kubectl get pods
+
+In order to check whether the ConfigMap and Secrets volumes are mounted correctly:
+
```

Comment:
There is a missing command or explanation here for checking if the ConfigMap and Secrets volumes are mounted correctly. Consider adding a command or description to guide the user.